docs/docs/faucet.md

```diff
@@ -7,3 +7,46 @@ Token Faucets are network specific, depending on the network type they may or ma
 The Token Faucet itself is available from the network block explorer (`GET FUNDS` button on the homepage).
```

Comment:
Now that we have additional faucet APIs for mobx and ulrn, is it worth mentioning that the block explorer faucet only provides atestfet?

Reply:
Hey @ianharris-fetch , I am assuming you are talking about the [networks](https://docs.fetch.ai/ledger_v2/live-networks/) page. 
On the network page, in the testnet section, we mentioned only fetch testnet URLs (RPC, GRPC and REST endpoint).